### ISSUE TITLE
[BUG] Game data deletion, Graphics corruption, and save data corruption in Doki Doki Literature Club! (698780)

### ISSUE THREAD

**Issue by nstgc** [ValveSoftware/Proton#4910]
This contains spoilers. Also disturbing imagery.

Thread's tl;dr: It's a feature of the game, not a bug.

# Compatibility Report
- Name of the game with compatibility issues: Doki Doki Literature Club! 
- Steam AppID of the game: 698780

## System Information
- GPU: Radeon RX5600
- Driver/LLVM version: 
- Kernel version: 
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/nstgc/2cc37b4dda2d55eb4f0ceb3f8435f675
- Proton version: 3.7, 6.3 plus GE's 6.9-1 and one other official Proton version I can't recall.

## I confirm:
I was told to start a new report by @kisak-valve

## Symptoms 

The first attempt at making a log was with Proton 6.3. The game didn't launch and the log file was created with basically nothing, just the stuff between the initial line of "===="s. After verifying the local files, I was told 106 files needed to be redownloaded. I had just exited out of the game after playing with `./DDLC.sh # %command%`, so I know the files were there.

I changed to Proton 3.7 and got the following unwelcomed sights.

![Screenshot from 2021-06-20 11-22-19](https://user-images.githubusercontent.com/25557355/122679798-406dec00-d1ba-11eb-84bf-3fa4634b9688.png)

![Screenshot from 2021-06-20 09-42-07](https://user-images.githubusercontent.com/25557355/122676876-e6ffc000-d1ad-11eb-99e1-6b1a196155b7.png)

[Log file can be found here](https://gist.github.com/nstgc/1861bd3c1d415fdd71aae6c44f3f8cfd)

I again verified the game data. This time 108 files needed to be redownloaded. Admittedly, my eye sight isn't great, so it's possible that I had previously misread the number earlier as 106 when really it was 108.

In an attempt to verify the number, I switched back to 6.3 and tried again. This time all files verified.

I then tied Proton GE-6.9-1 with the same result as 3.7 including having to redownload 108 files.

At this time, I can't reproduce the issue that kicked this whole thing off which may be seen in the following screen shot.

![Screenshot from 2021-06-20 09-17-48](https://user-images.githubusercontent.com/25557355/122677056-92a91000-d1ae-11eb-9ed8-010625d96400.png)

I also can't reproduce the issue with the save game corruption. For this I'd need to be able to run the game in Proton properly first, then switch to another version. I have a backup of the game directory which I could try restoring, however that will have to wait. This is the first Father's Day since my father passed and I've already exceeded the limits of what I can do today, so this will have to do for now. I'll fill out the rest at a later date, but I was told to make a new thread and this is that thread.

I'm also attaching the original post I made.

## Reproduction

Does not work with Proton 6.3. Some text won't load at a certain point (which I'll get to later since it's a spoiler... sort of). When I tried changing to a different version of proton everything went crazy.

The first thing that happened was that some of the main menu text was gibberish (Part of another screen shot). Not that it was in a foriegn language—most of it was still in English—but that is was just messed up. When I went to load a save, it said something was corrupted and some other stuff (should have taken a screenshot). Then the save was gone... sort of.

So I exit the game and try again. It's at this point that Sayori on the main menu is messed up. Her sprite is some unholy amalgamation of all four of the girls in box form.
![Screenshot from 2021-06-20 09-42-07](https://user-images.githubusercontent.com/25557355/122676876-e6ffc000-d1ad-11eb-99e1-6b1a196155b7.png)
This also illustrates the messed up text I menioned earlier.

It's at this point I verify the game data and find that over 100 files need to be replaced. So I do that and try again. I'm told that an old save was found and I'm asked if I want to delete it. I say "no". That helped, not at all.

Rinse and repeat a few times with different versions. For some reason, even the version that's default (nothing is ticked in the compatibility menu) is borked.

Eventually I try launching from DDLC.sh. The game seems to work, but my saves are gone. Which really sucks since I'd very much like to know what the text on the wall is in the screenshot below.

Spoiler warning (though at this point I'm pretty sure everyone is basically waiting for it).

![Screenshot from 2021-06-20 09-17-48](https://user-images.githubusercontent.com/25557355/122677056-92a91000-d1ae-11eb-9ed8-010625d96400.png)

I must admit, though, that an "An exception has occurred" is pretty apropos. The traceback at this point is basically gone, sadly. I should have looked at that first.

[Steam System Info](https://gist.github.com/nstgc/2cc37b4dda2d55eb4f0ceb3f8435f675)

**Comment by nstgc** [comment]
I don' think this is a Proton bug. I continued playing it with `./DDLC.sh # %command%` as the optional command. I got the same exception error. I exit the game. When I start the game up again, things are a mess. The only difference is when I run the game data verifier, 107 files need to be redownloaded.

I deleted the game directory, uninstalled the game, then redownloaded it. I try loading the old save. I get the following.

![Screenshot from 2021-06-20 14-31-23](https://user-images.githubusercontent.com/25557355/122684691-e4649100-d1d4-11eb-98fb-f79a46f329c2.png)

Then a message about starting a new game.

![Screenshot from 2021-06-20 14-31-39](https://user-images.githubusercontent.com/25557355/122684693-e9294500-d1d4-11eb-97bf-089af0ca2fea.png)

So I delete the directory again, and download the game from Itch.io and played (set to auto) without Steam or Wine having anything to do with the game, making the same sort of choices before. I also let the game delete all my saves so I can start fresh. I should have deleted the ~/.renpy/DDLC-whatever directory too, but didn't.

I get the same exception error. This time, however, I decided to just continue on. I really wish I could have seen whatever that text was, but whatever.

When I'm dumped back to the main menu, everything is messed up just like before.

So whatever is going on is a problem with the game itself and not Proton.
